mypy.ini

```diff
@@ -0,0 +1,2 @@
+[mypy]
+ignore_missing_imports = true
```

Comment:
Why isn't this config in the respective `pyproject.toml` files?

Reply:
Updated the makefile command to `cd <plugin` and run `mypy` to use config from `pyproject.toml`

Reply:
Reverting to top level config - `mypy` isn't picking up settings from individual `pyproject.toml`